action.yml

```diff
@@ -0,0 +1,37 @@
+name: 'Changelog included'
+description: 'Verify if a new changelog is included'
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/checkout@v2
+      with:
+          fetch-depth: 2
+    - name: Check changelog file added
```

Comment:
The style we follow in other repos is to leave one empty line between steps blocks so this would be something like
```
- name: step1
  run: ...

- name: step2
  run: ...

- name: stepn
  run: ...
```